prac_07/project_management.py

```diff
@@ -4,45 +4,96 @@
 Estimate: minutes
 Actual: minutes
 """
+import datetime
+
 from project import Project
 
 
 def main():
     projects = []
     print_menu()
     choice = input(">>> ").upper()
+
     while choice != "Q":
-        # if choice == "L":
-        # in_filename = input("Data file to load from: ")
         projects = load_data("projects.txt", projects)
         # print("Data loaded.")
+        if choice == "L":
+            filename = input("Filename: ")
+            print("Projects loaded")
+
         if choice == "S":
             out_filename = input("Data file to save to: ")
             write_data(out_filename, projects)
-            print("Data saved.")
+            print("Projects saved.")
+
         if choice == "D":
             display_projects(projects)
+
         if choice == "U":
             for project in projects:
                 print(f"{projects.index(project)} {project}")
-            project_choice = input("Project Choice: ")
+            project_choice = int(input("Project Choice: "))
             print(projects[project_choice])
-            new_percentage = int(input("New Percentage: "))
+            new_percentage = input("New Percentage: ")
+            get_new_percentage(new_percentage, project_choice, projects)
+            new_priority = input("New Priority: ")
+            get_new_priority(new_priority, project_choice, projects)
+
+        if choice == "A":
+            print("Let's ass a new project")
```

Comment:
Maybe just check the spelling here.
I doubt we want to "ass" a new project
But maybe we want to add it (insert laughing emoji here)